datadog/tests/resource_datadog_dashboard_geomap_test.go

```diff
@@ -0,0 +1,100 @@
+package test
+
+import (
+	"testing"
+)
+
+const datadogDashboardGeomapConfig = `
+resource "datadog_dashboard" "geomap_dashboard" {
```

Comment:
let's add a `custom_link` field in the tests too

Reply:
Done in commit [cc356af](https://github.com/DataDog/terraform-provider-datadog/pull/954/commits/cc356af8aa7b6b51bb3dbb0c4ec4bbd2b707f181).